test/net/sourceforge/kolmafia/maximizer/MaximizerTest.java

```diff
@@ -120,7 +122,7 @@ public void freshCharacterShouldNotRecommendEverythingWithCurrentScore() {
   @Test
   public void noTieCanLeaveSlotsEmpty() {
     // 1 helmet turtle.
```

Comment:
nit: this comment is no longer useful now that you've removed loadInventory.

(ditto above in a few other situations)